src/main/interface/settings.c

```diff
@@ -714,6 +714,9 @@ const clivalue_t valueTable[] = {
     { "pidsum_limit_yaw",           VAR_UINT16 | PROFILE_VALUE, .config.minmax = { 100, 1000 }, PG_PID_PROFILE, offsetof(pidProfile_t, pidSumLimitYaw) },
     { "yaw_lowpass_hz",             VAR_UINT16 | PROFILE_VALUE, .config.minmax = { 0, 500 }, PG_PID_PROFILE, offsetof(pidProfile_t, yaw_lowpass_hz) },
 
+    { "throttle_boost",             VAR_UINT8 | PROFILE_VALUE,  .config.minmax = { 0, 100 }, PG_PID_PROFILE, offsetof(pidProfile_t, throttle_boost) },
+    { "throttle_boost_cutoff",      VAR_UINT8 | PROFILE_VALUE,  .config.minmax = { 10, 50 }, PG_PID_PROFILE, offsetof(pidProfile_t, throttle_boost_cutoff) },
```

Comment:
Is 10Hz low enough(no idea)? Maybe it's good idea to allow it too low to show that too low cutoff has negative influence on performance ...

Reply:
I think it's low enough. SilverWare uses 15hz and that's primarily for whoop style brushed quads. But let me make it 5 as minimum - that should absolutely suffice.